book/3-classification.tex

```diff
@@ -268,4 +268,74 @@ \subsection{Phi Coefficient}
 \clearpage
 \thispagestyle{classificationstyle}
 \section{MCC}
-\subsection{Matthew's Correlation Coefficient}
\ No newline at end of file
+\subsection{Matthew's Correlation Coefficient}
+
+
+% ---------- EC ----------
+\clearpage
+\thispagestyle{classificationstyle}
+\section{EC}
+\subsection{Expected Cost}
+
+The Expected Cost (EC) metric is designed to evaluate the performance of decision-making systems. It is the expected value of a cost function that assigns a penalty to each combination of the system's decision and the true class of a sample.
+
+\begin{center}
+\tikz{
+\node[inner sep=2pt, font=\Large] (a) {
+{
+$\displaystyle
+EC = \frac{1}{{\color{red}N}} \sum_{t=1}^{N} {\color{purple}C}({\color{cyan}h_t}, {\color{orange}d_t})
```

Comment:
Could you modify the formula so it is in terms of Y instead of $h$ and Y_hat intead of $d$, so we keep that standardization for most of the book metrics?

Reply:
Yup sure, but d here specifically refers to the decisions, which is not necessarily the part of h or (Y.)
![image](https://github.com/user-attachments/assets/7169525f-b7c0-4012-8c83-3ab7c4f3be1a)

Also I am planning to include other form of the formula.
![image](https://github.com/user-attachments/assets/3fa0da43-6b1a-4fdf-a9cb-cc77c1000f8f)

Reply:
oh, cool, that particular explanation would be a great addition.

"EC does not assume that the set of possible decisions is the set of classes. Instead, the decisions can be the actions that may be taken by the user of the system."

I'm not sure about the other formula form, but it could be nice. Looking at the cost matrix example that you made on the second page, I would rather have a formula explaining how to calculate the EC from that table.

Reply:
"I would rather have a formula explaining how to calculate the EC from that table."
Yup that's exactly the next step I am planning.